cfme/networks/views.py

```diff
@@ -518,6 +519,17 @@ class NetworkRouterAddInterfaceView(BaseLoggedInPage):
     is_displayed = displayed_not_implemented
 
 
+class NetworkRouterAddCloudSubnetView(BaseLoggedInPage):
+    """ Represents Add Interface to Network Router page """
+    subnet_name = TextInput(id='name')
+    address = TextInput(id='address')
+    netmask = TextInput(id='netmask')
+    gateway = TextInput(id='gateway')
+    add = Button('Add')
+
+    is_displayed = displayed_not_implemented
```

Comment:
Same here, I was following pattern of the other edit/add popup views. I checked the code and I couldn't find any popups with the `is_displayed` method implemented, maybe because it's a popup and doesn't work like the other pages...

Reply:
@aljazkosir here also I'm agreed with you... but what you think we should implement more robust  implementation or not? I would like to add `is_displayed` with some unique checks. so it will help us to improve checks. 

https://github.com/ManageIQ/integration_tests/pull/8254/files#diff-78b559a596dffca17c9934f224cafa47R244

To help navmazing we should write more unique view checks.

Reply:
Yes, I think it would be better to have some unique checks. I added is_displayed function, which checks if module title is "Create L3 Cloud Subnet'".

Reply:
The reason, AFAIK, that these views are specifically not using is_displayed is because of the interconnected nature of the collections and entities. Just checking the title here is not sufficient, as this is a cloud subnet tied to a specific Network Router entity.

Sometimes adding is_displayed makes the framework _less_ robust, as the check is insufficient and produces false positives - making for strange test failures or behavior, or even worse a false positive test result.